lua/bookmarks/domain/bookmark_list.lua

```diff
@@ -33,9 +33,16 @@ end
 function M.find_bookmark_by_location(self, location)
   -- TODO: self location path
   for _, b in ipairs(self.bookmarks) do
+    local b_type = M.get_value_type(b)
```

Comment:
Seems like this commit will fail the sign and line highlight.
And also the original pick method is not working as well

Reply:
is there any exception